server/src/main/java/org/opensearch/threadpool/ThreadPoolStats.java

```diff
@@ -125,6 +135,10 @@ public long getCompleted() {
             return this.completed;
         }
 
+        public TimeValue getWaitTimeNanos() {
```

Comment:
It should really return just long, as it says, "wait time nanos":

```suggestion
        public long getWaitTimeNanos() {
```